src/css/components/trigger.scss

```diff
@@ -0,0 +1,74 @@
+@import '../core.scss';
+
+.usa-button.trigger {
```

Comment:
I'm having trouble understanding what `button trigger` is even after viewing styleguide. It looks like a sort of drop down button? 

I also think the styleguide should be consistent and should be called trigger.